packages/components/src/pages/quickstart-docs/InstallAgent.vue

```diff
@@ -32,6 +35,14 @@ export default {
     VCodeSnippet,
   },
 
+  data() {
+    return {
+      codeSnippet: 'npx @appland/appmap install-agent',
+      messageSuccess:
+        '<b>Copied!</b><br>Paste into your terminal<br>to run the installer',
```

Comment:
In the storybook, the success message just says 'Copied to clipboard.' Is there some way I can see the full text of this success message? 
![image](https://user-images.githubusercontent.com/1229326/131927317-1faa25db-84b4-4288-a4ed-31496c49f373.png)